quarkchain/config.py

```diff
@@ -110,6 +110,13 @@ class POWConfig(BaseConfig):
     REMOTE_MINE = False
 
 
+class POSWConfig(BaseConfig):
+    STAKE_COEFF = 20  # Alpha in PoSW
```

Comment:
I think STAKE_PER_BLOCK may be easier

Reply:
discussed offline, will combine alpha and total stakes to a single parameter